### PR TITLE
Add option for addiing reviewers from original pr

### PR DESCRIPTION
**Pull request by reey**

### CODE REVIEW

**Comment by sqren** [comment]
Hi there

Thanks for the contribution. I think this looks great but I'd rather add it to the underlying package https://github.com/sqren/backport. Feel free to take a stab at it or I'll take a look when I find some free time.

**Comment by reey** [comment]
@sqren I've placed it here in the action since it is already done the same way for the assignee.
As far as I know the context of the origin PR is also missing for the backport tool and just available within this action..

File: src/run.ts [hunk 1]
```diff
@@ -31,13 +32,23 @@ export async function run({
...
   const assignees = [pullRequest.user.login];
+  let reviewers: string[] | undefined = undefined;
+  if (
+    inputs.addOriginalReviewers &&
+    Array.isArray(pullRequest.requested_reviewers)
+  ) {
+    reviewers = pullRequest.requested_reviewers.map(
+      (reviewer) => reviewer.login
+    );
+  }
```
Comment (sqren) [inline src/run.ts:43]:
```suggestion
  const requestedReviewers = pullRequest.requested_reviewers as
    | Array<{ login: string }>
    | undefined;

  const reviewers =
    inputs.addOriginalReviewers !== '' && Array.isArray(requestedReviewers)
      ? requestedReviewers.map((reviewer) => reviewer.login)
      : [];
```

**Review by sqren — APPROVED** [review]
Looks good to me. Just one smaller comment. Thanks for creating this!

**Comment by sqren** [comment]
Released in https://github.com/sqren/backport-github-action/releases/tag/v8.9.5

**Pull request merged**